Putting Hair On Them Sharks 'And Lizards'

ArbyTheDragon opened the pull request:
A hopefully not in vain attempt at giving resident sharks and lizards the hairstyles they aught to have.

ArbyTheDragon committed:
Putting Hair On Them Sharks 'And Lizards'

A hopefully not in vain attempt at giving resident sharks and lizards the hairstyles they aught to have.

ShadowLarkens commented on code/modules/mob/new_player/sprite_accessories.dm:
```diff
@@ -46,6 +46,7 @@
 /datum/sprite_accessory/hair
 
 	icon = 'icons/mob/Human_face.dmi'	  // default icon for all hairs
+species_allowed = list("Human","Unathi","Akula")
```
this tabbing is broken

ArbyTheDragon replied:
Oops. Probably. Well, there was an attempt.

ArbyTheDragon replied:
It IS broken. And I fixed it. I just need to edit my commit.

ArbyTheDragon committed:
(Fixed) Putting Hair On Them Sharks 'And Lizards'

A fixed version of my last pull request to allow hairstyles on humans, Akula, and Unathi unless otherwise stated.
Thanks to ShadowLarkens for telling me my tabbing is dumb

I made this edit and compiled it myself in a build I have sitting on my computer to make sure what I was doing is correct.
And it works.

ArbyTheDragon commented:
It's fixed, I wrote it to a testing build on my computer, and **it works**.
